pkg/apis/servicecatalog/types.go

```diff
@@ -774,6 +774,10 @@ type ServiceInstanceSpec struct {
 	// allows for parameters to be updated with any out-of-band changes that have
 	// been made to the secrets from which the parameters are sourced.
 	UpdateRequests int64
+
+	// DeletionRequested provides an imperative means to request
```

Comment:
The comment doesn't really make it clear why we are adding this way to delete or when someone would want to use it. Maybe something like this?

```
// Deletion requested provides a mechanism for requesting a delete without performing a direct 
// delete (which cannot be undone in cases where the delete is rejected by the broker). This is the 
// recommended way to delete, to avoid resources becoming stuck in the deletion phase.
```

Reply:
Ya, you betcha.

At this point I want to establish the scaffolding to build everything on. Since openapi uses these comments, they do become documentation.